package.json

```diff
@@ -56,6 +56,7 @@
     "logops": "1.0.0-alpha.7",
     "mqtt": "1.7.0",
     "request": "^2.69.0",
+    "mongodb": "2.2.10",
     "underscore": "1.8.3"
   }
 }
```

Comment:
Maybe is a good idea to regenerate npm-shrinkwrap.json each time we touch package.json. The procedure is easy:

1. Remove npm-shrinkwrap.json and node_moduler dir
2. `nmp install`
3. `npm npm-shrinkwrap.json`